### ISSUE TITLE
Superscript characters from wikipedia.org are always displayed at the right side of the string it's associated to

### ISSUE THREAD

**Issue by avaida** [mozilla/readability#107]
**Reproducible on**: 
Beta 38.0b1-build1 (20150330154247)

**Screenshot**:
http://i.imgur.com/sSox0Yt.png

**Sample page showing this issue**:
http://en.wikipedia.org/wiki/Sonic_X

**Comment by gijsk** [comment]
@avaida I don't understand the summary of this issue and the screenshot. What is wrong, according to you?

Testing this is made more difficult because on my current Nightly, Wikipedia doesn't work at all... :-\

**Comment by gijsk** [comment]
Filed #138 for the complete failure here.

**Comment by avaida** [comment]
@gijsk Take a look at the References section from the screenshot - the "a" and "b" characters styled as superscript (e.g. from item 1. in that list) are displayed at the left side in the original article, while in Reader View they're displayed at the right side. This happens for every wiki page, it places the superscript at the right side at all times.

**Comment by gijsk** [comment]
@avaida Yeah, this is what I don't understand. AFAICT they are still displayed where they should be - it's just that the "jump up to" text gets inserted before it. It's presumably a tooltip of the "^" thing that somehow gets inserted. Right?

**Comment by gijsk** [comment]
Ah, it's a:

`<span class="cite-accessibility-label">Jump up to: </span>`

label which is in the markup. I don't know that we should do something about that, though we could relatively easily add it into our site-specific CSS hack lists... @leibovic , thoughts?
